site/devtools.md

```diff
@@ -194,8 +193,8 @@ Miscellaneous projects:
 
 ## <a id="erlang-dev" class="anchor" href="#erlang-dev">Erlang</a>
 
- * [RabbitMQ Erlang client guide](erlang-client-user-guide.html)
- * [RabbitMQ Stream Erlang client](https://gitlab.com/evnu/lake)
+ * &#x2713; [RabbitMQ Erlang client](erlang-client-user-guide.html)
+ * &#x2713; [RabbitMQ Stream Erlang client](https://gitlab.com/evnu/lake)
```

Comment:
This Client doesn't seem official since its copyrights and organization don't seem to be related with the RabbitMQ.

Reply:
You're correct, please feel free to open a PR to remove the checkmark.